firebase.config.js

```diff
@@ -0,0 +1,33 @@
+import { initializeApp } from 'firebase/app';
```

Comment:
こうしてみて動作確認をしてみてください！
```suggestion
import { initializeApp } from "@firebase/app";
```